jobs/grafana/templates/post-start

```diff
@@ -5,3 +5,11 @@ $(dirname $0)/create-update-datasources
 
 echo "[$(date)] Calling 'create-update-dashboards' ..."
 $(dirname $0)/create-update-dashboards
+
+echo "[$(date)] Updating password for admin user"
+(
```

Comment:
Using a subshell here will prevent a failure within from making the outer script fail. Is that intentional?

May want to use `pushd` and `popd` instead, if not.